src/app/components/WorkDetail/WorkDetail.jsx

```diff
@@ -117,72 +95,76 @@ class WorkDetail extends React.Component {
           />
           <div className="grid-row">
             <div className="sfr-center">
-              <SearchForm
+              <SearchHeader
                 isHomePage={false}
                 history={history}
                 {...this.boundActions}
               />
             </div>
           </div>
-          <div className="grid-row">
-            <div className="nypl-item-header">
-              <WorkHeader data={work} />
+          { isValidWork
+          && (
+          <>
```

Comment:
We used to use `divs` because there wasn't a choice - but ideally, we don't want to nest our code in divs that don't do anything.  Since we're using React 16, might as well make use of this feature!